translate/storage/test_mozilla_lang.py

```diff
@@ -9,6 +9,21 @@
 from translate.storage import mozilla_lang, test_base
 
 
+@pytest.mark.parametrize(
+    "orig, stripped", [
+        ("", ""),
+        ("String", "String"),
+        ("String {ok}", "String"),
+        ("String {OK}", "String"),
+        ("String{ok}", "String"),
+        ("String{OK}", "String"),
+        ("String  {ok}", "String"),
```

Comment:
Its not missing, its testing something else i.e. two leading spaces, I've added comments.

I'll add one for the trailing space concern.